src/core_ocean/mode_init/mpas_ocn_init_tidal_boundary.F

```diff
@@ -337,6 +408,7 @@ subroutine ocn_init_setup_tidal_boundary(domain, iErr)!{{{
       if (config_use_wetting_drying .and. config_tidal_start_dry) then
         do iCell = 1, nCellsSolve
           ssh(iCell) = -bottomDepth(iCell) + config_drying_min_cell_height*maxLevelCell(iCell)
+          ssh(iCell) = MAX(ssh(iCell),-config_tidal_forcing_monochromatic_baseline) !Zhendong Mar-06-2020
```

Comment:
@caozd999, can you please explain the purpose here?

Reply:
In **shared/mpas_ocn_tidal_forcing.F**, the tidalHeight is computed as
>`tidalHeight = config_tidal_forcing_monochromatic_amp * &
             SIN(2.0_RKIND*pii/config_tidal_forcing_monochromatic_period *daysSinceStartOfSim - & pii*config_tidal_forcing_monochromatic_phaseLag/180.0_RKIND) - &
             config_tidal_forcing_monochromatic_baseline`

So adding the line 
>`ssh(iCell) = MAX(ssh(iCell),-config_tidal_forcing_monochromatic_baseline)` 

will ensure that tides propagate just on top of the initial ssh, instead of possibly having a distance above the initial ssh.